Categories/Morphism/Isomorphism.agda

```diff
@@ -56,6 +56,9 @@ private
     }
     where open _≅_ f
 
+-- sstucki: shouldn't this go into IsoEquiv together with the
```

Comment:
This might be an artifact of porting the old library. And I don't mind splitting some things off into smaller files. I think there was also a circular import problem lurking around here too, but I'm not entirely sure anymore.